Resources/Textures/Structures/Walls/vault.rsi/meta.json

```
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/e09101cb4be860ff02e035b18bef93a08b9c204c modified by Peptide",
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/e09101cb4be860ff02e035b18bef93a08b9c204c modified by Peptide, 'alienvault' resprited by Ko4erga (discord)",
```

лучше наши копирайты делать во вторую строчку, легче при апстриме
```suggestion
  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/e09101cb4be860ff02e035b18bef93a08b9c204c modified by Peptide",
  "copyright": "твоя хуйня",
```

---

не может быть два ключа одиенаковых в одном файле

---

может